contrib/pgo-lto/Makefile

```diff
@@ -0,0 +1,57 @@
+.PHONY: top clean
+
+STAGE1_BUILD:=$(CURDIR)/stage1.build
+STAGE2_BUILD:=$(CURDIR)/stage2.build
+
+PROFILE_DIR:=$(CURDIR)/profiles
+PROFILE_FILE:=$(PROFILE_DIR)/merged.prof
+JULIA_ROOT:=$(CURDIR)/../..
+CXXFILT:=c++filt
+
+# Note: can't use BB's LLVM right now; -fprofile-generate compiles, but doesn't
+# generate profiles at runtime
+LLVM_PROFDATA:=llvm-profdata
+export CC:=clang
+export CXX:=clang++
+export LD:=ld.lld
+export AR:=llvm-ar
+export RANLIB:=llvm-ranlib
+export USECLANG:=1
+export USE_BINARYBUILDER_LLVM:=0
+# export VERBOSE:=1
+
+# When building a single libLLVM.so we need to increase -vp-counters-per-site
+# significantly
+COUNTERS_PER_SITE:=6
+
+AFTER_STAGE1_MESSAGE:=Consider removing $(PROFILE_DIR) to start with a clean slate. $\
+    Then run Julia to collect realistic profile data, for example: \`julia -O3 -e $\
+    'using Pkg; Pkg.add("Unitful"); Pkg.test("Unitful")'\`. This should produce about $\
+	15MB of data in $(PROFILE_DIR). Note that running extensive scripts may result in $\
+	counter overflows, which can be detected by running \`make $(PROFILE_FILE)\`. $\
+	Afterwards run `make stage2`.
```

Comment:
It sounds here like you must run `make profiles/merged.prof` first, then `make stage2`.  But reading the `Makefile`, it sounds like `make stage2` will do that for you automatically.